src/tests/components/App.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import App from '../../components/App';
+describe('App', () => {
+  it('div tag displays correctly in "debug" mode', () => {
+    const component = shallow(<App debug />);
+    expect(component.find('div')).toBeDefined();
+  
+  });
+  it('H3 tag displays correctly in "debug" mode', () => {
+    const component = shallow(<App debug />);
+    expect(component.find('div.h3')).toBeDefined();
+  
+  });
+});
```

Comment:
@BrianSerem Please add an end of file to this file too.

Reply:
@Etomovich the files you have cited EOLs

Reply:
Etole this file has an end of line, files without have little red arrows at the bottom

Reply:
This is now ok with me.